pkg/util/runner.go

```diff
@@ -33,6 +33,7 @@ func (r RealRunner) Run(command string, args ...string) error {
 	fmt.Println(fmt.Sprintf("Provisioning OpenShift via '%s %s'", command, args))
 	cmd := exec.Command(command, args...)
 	cmd.Stdout = os.Stdout
+	cmd.Stderr = os.Stdout
```

Comment:
> Provisioning OpenShift via '/Users/prkumar/.minishift/cache/oc/v1.4.1/oc [cluster up --use-existing-config --host-config-dir /var/lib/minishift/openshift.local.config --host-data-dir /var/lib/minishift/hostdata --http-proxy http://fedora:fedora123@10.70.49.109:3128]'
Error: unknown flag: --http-proxy

I think we should do better than that and have a nicer error message. There is a lot of 'oc cluster' in there which a novice might not understand. Let's do the mapping approach and determine beforehand whether we can pass the proxy. If not, we can say something along the line:

"The requested Openshift version <version> does not support HTTP proxies. You will need to use a newer version of OpenSift".

BTW, do we want to halt the VM at this stage? I guess it is running right?